files/mono-build-win32.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
 
-echo -e '#!/bin/bash\nwine${WINE_BITS} $(dirname $0)/mono-sgen.exe "$@"' > mono/mini/mono
+if [ "${WINE_BITS}" == "64" ]; then
+  export WINEPATH="/usr/x86_64-w64-mingw32/sys-root/mingw/bin/"
+else
+  export WINEPATH="/usr/i686-w64-mingw32/sys-root/mingw/bin/"
```

Comment:
This was sufficient to fix the 32-bit build. I also added the 64-bit variant for consistency, just in case.